pom.xml

```diff
@@ -403,6 +403,26 @@ under the License.
               </rules>
             </configuration>
           </execution>
+          <execution>
+            <id>enforce-maven.compiler.target-version</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireProperty>
+                  <property>maven.compiler.target</property>
+                  <regex>(?!1\.).*</regex>
+                  <regexMessage>The value of property maven.compiler.target must not start with "1.", use for example "8" instead of "1.8"!</regexMessage>
+                </requireProperty>
+                <requireProperty>
+                  <property>maven.compiler.source</property>
+                  <regex>(?!1\.).*</regex>
+                  <regexMessage>The value of property maven.compiler.source must not start with "1.", use for example "8" instead of "1.8"!</regexMessage>
+                </requireProperty>
+              </rules>
+            </configuration>
+          </execution>
```

Comment:
I would like to not add next execution, simply if user has wrong value build will fail.

Reply:
I'll let it fail implicitly...

Reply:
I removed the enforcement of the property in https://github.com/apache/maven-apache-parent/pull/188/commits/8ab37cef72208e2b9667dbd298a446b16696ec66.